### PR TITLE
Rust cleanups

### PR DESCRIPTION
**Pull request by nazar-pc**
This brings code quality to a bit higher standard, adds some preludes with WebRtcServer for convenience and removes previously deprecated method.

Nothing major overall.

### CODE REVIEW

File: rust/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
   * usrsctp snapshot 4e06feb01cadcd127d119486b98a4bd3d64aa1e7.
   * wingetopt 1.00.
 * `WebRtcServer`: A new class that brings to `WebRtcTransports` the ability to listen on a single UDP/TCP port (PR #834, PR #845).
+* Minor API breaking changes
```
Comment (ibc) [inline rust/CHANGELOG.md:12]: Probably a missing dot.

**Review by ibc — APPROVED** [review]

File: rust/CHANGELOG.md (hunk 1 above)
Comment (nazar-pc) [reply rust/CHANGELOG.md:12]: This changelog is inconsistent with dots already, probably not worth it :upside_down_face:, I wish there was a linter that could insert or remove them automatically in consistent way.

**Comment by nazar-pc** [comment]
CI failed in my fork with segfault: https://github.com/nazar-pc/mediasoup/runs/7164088872?check_suite_focus=true
Not related to changes in this PR, but there must be something somewhere that is wrong, will have to investigate.

File: rust/CHANGELOG.md (hunk 1 above)
Comment (ibc) [reply rust/CHANGELOG.md:12]: I never found that rule 🙁

**Pull request merged**